src/main_menu.cpp

```diff
@@ -348,14 +351,13 @@ void main_menu::print_menu( const catacurses::window &w_open, int iSel, const po
 
     int menu_length = 0;
     for( size_t i = 0; i < vMenuItems.size(); ++i ) {
-        menu_length += utf8_width_notags( vMenuItems[i].c_str() ) + 2;
-        if( !vMenuHotkeys[i].empty() ) {
-            menu_length += utf8_width( vMenuHotkeys[i][0] );
-        }
+        const std::string shortcut = remove_color_tags( shortcut_text( vMenuItems[i] ) );
+        const int shortcut_width = utf8_width_notags( shortcut.c_str() ) + 1;
```

Comment:
The following might fix the centering problem:
```suggestion
        const int shortcut_width = utf8_width_notags( shortcut.c_str() );
```

Reply:
that combined with your other fix did the trick

![image](https://user-images.githubusercontent.com/2420411/228954437-7e5c4020-bf17-4451-bbd1-e81e79a7600b.png)


it works! ;)